packages/ui/cypress/support/commands.js

```diff
@@ -26,7 +26,7 @@
 
 import 'cypress-testing-library/add-commands'
 
-Cypress.Commands.add('login', (username, password) => {
+Cypress.Commands.add('login', (username, password, failOnStatusCode) => {
```

Comment:
can you refactor this with an object as argument? it's difficult to know the order of three arguments, I think it's better like this:
```
({
  username,
  password,
  failOnStatusCode = true,
} = {}) =>
```